### ISSUE TITLE
cloud plugin breaks when test files have an error

### ISSUE THREAD

**Issue by reimakesgames** [tacheometry/testez-companion#11]
Context:
why a cloud plugin? the vscode extension is having trouble moving the plugin file into my plugins file for roblox studio, so I've decided to put it on the cloud instead.

Issue:
The local plugin throws an error when there is an error in my .spec files, and then it doesn't respond to other test calls afterward.

**Comment by reimakesgames** [comment]
i've made a temporary fix by replacing the lines in the plugin script around line 90 to
```lua
...
hotReload.flush()
local success, err = pcall(function()
	local TestEZ = hotReload.require(script.Parent.TestEZ)
	TestEZ.TestBootstrap:run(roots, reporter, config.testExtraOptions)
end)
if not success then
	warn("An error has occurred!")
	warn(err)
end
logServiceConnection:Disconnect()
...
```

although it's insufficient and apparently it doesn't print the error anymore.

**Comment by tacheometry** [comment]
I don't really know what a cloud plugin is. I think that is what is causing the error. Errors should be caught by TestEZ, and reported. On my install, with a local plugin, having a test like this:

```lua
return function()
    describe("test", function()
        it("should error and fail", function()
            error("Hello world")
        end)
    end)
end
```

Causes it to fail, show up properly in the explorer, and report in the Output menu:

```
[🔴ERROR]
[string "ReplicatedStorage.Common.tests.spec"]:5: Hello world
[string "ReplicatedStorage.Common.tests.spec"]:5
```

I'm also able to run more tests - the Studio side plugin does not crash.

I think this is related to your issue, #12?

**Comment by tacheometry** [comment]
Nevermind, I was testing with the wrong code. Should be fixed in 2.1.2.
